kms/api/CloudKmsSample/CloudKmsSample.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.3.0" />
     <PackageReference Include="Google.Cloud.Kms.V1" Version="1.0.0" />
+    <PackageReference Include="Portable.BouncyCastle" Version="1.8.5.2" />
```

Comment:
I'm not crazy about the BC dependency, but System.Security.Cryptography doesn't get the crypto we need in .NET Core until 3.0 :(

Reply:
As I understand it, it's not just a library change.  It means upgrading the project's `targetFramework` from `netcoreapp2.0` to something later, since ultimately the runtime needs to be able to invoke the right OpenSSL APIs.